client/blocks/edit-gravatar/index.jsx

```diff
@@ -238,29 +236,62 @@ export class EditGravatar extends Component {
 					{ 'is-unverified': ! user.email_verified },
 					{ 'is-uploading': isUploading }
 				) }
+				role="region"
+				aria-label={ translate( 'Profile photo editor' ) }
 			>
 				<div onClick={ this.handleUnverifiedUserClick }>
-					<FilePicker accept="image/*" onPick={ this.onReceiveFile }>
+					<FilePicker
+						accept="image/*"
+						onPick={ this.onReceiveFile }
+						aria-describedby="gravatar-description gravatar-instructions"
+					>
 						<div
 							data-tip-target="edit-gravatar"
 							className={ clsx( 'edit-gravatar__image-container', {
 								'is-uploading': isUploading,
 							} ) }
+							role="button"
```

Comment:
Isn't the wrapper `FilePicker` div the actual button? It feels like we might need to push some of the accessibility items into that component rather than trying to add them all here.

Reply:
Ah yes, you're correct. I misunderstood where the click was happening. I'll move those attributes.